7alip_week2/strings-arrays/app.js

```diff
@@ -0,0 +1,64 @@
+function stringsChallenge() {
+  let myString = "hello,this,is,a,difficult,to,read,sentence";
+  const lengthStr = myString.length;
+  console.log(myString); // hello,this,is,a,difficult,to,read,sentence
+
+  console.log(myString.length); // 42
+
+  let modifiedStr = myString.split(',').join(' ');
+  // let modifiedStr = myString.replace(/,/g, ' ');
+  console.log(modifiedStr); // hello this is a difficult to read sentence
+}
+
+function arraysChallenge() {
+  let favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+  let maurosAnimal = 'turtle';
+  favoriteAnimals.push(maurosAnimal);
+
+  console.log(favoriteAnimals); // [ 'blowfish', 'capricorn', 'giraffe', 'turtle' ]
+
+  /* 
+  let newFavoriteAnimals = favoriteAnimals.push(maurosAnimal);
+  console.log(newFavoriteAnimasls) // logs 4, not the array itslef
+
+  If you assign new pushed array to a new variable, it returns a number. if you want to log desired result, you need to call it without assigning. 
+  */
+
+  let jimsAnimal = 'meerkat';
+
+  // Here is the solution https://stackoverflow.com/a/12710609
+  Array.prototype.insert = function (index, item) {
+    this.splice(index, 0, item);
+  }
+
```

Comment:
This is a good solution, however, what this solution does is, it copies the original array and then edits that copied version, leaving the original array unchanged. For this homework you could just use splice(), it is faster and it directly changes the original array, which I think is what homework requires.